spark/src/main/scala/org/apache/spark/delta/sharing/PreSignedUrlCache.scala

```diff
@@ -127,20 +127,20 @@ class CachedTableManager(
    * @param tablePath the table path. This is usually the profile file path.
    * @param idToUrl the pre signed url map. This will be refreshed when the pre signed urls is going
    *                to expire.
-   * @param ref A weak reference which can be used to determine whether the cache is still
+   * @param refs A list of weak references which can be used to determine whether the cache is still
    *                  needed. When the weak reference returns null, we will remove the pre signed
    *                  url cache of this table form the cache.
```

Comment:
Consider updating the comment about whether all of them need to return null for them to be removed?

Reply:
updated.